transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodFragment.java

```diff
@@ -361,8 +362,18 @@ void bind(@NonNull final TransferMethod transferMethod) {
                 Locale locale = new Locale.Builder().setRegion(
                         transferMethod.getField(TRANSFER_METHOD_COUNTRY)).build();
                 mIcon.setText(getStringFontIcon(mIcon.getContext(), type));
-                mTransferMethodCountry.setText(locale.getDisplayName());
-                mTransferMethodIdentification.setText(transferMethodIdentification);
+                if(type.equals(PREPAID_CARD))
+                {
```

Comment:
Open `{` needs to be in line 365, please follow the current code convention

Reply:
fixed